<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_refresh">
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_connect">
     <property name="text">
      <string>连接读卡器</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_disconnect">
     <property name="text">
      <string>断开连接</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QComboBox" name="comboBox_readerList"/>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QPushButton" name="pushButton_transmit">
     <property name="text">
      <string>SCardTransmit</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QLabel" name="label_recv">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEdit_transmit"/>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEdit_control"/>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QPushButton" name="pushButton_control">
     <property name="text">
      <string>SCardControl</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
